napari/utils/settings/_manager.py

```diff
@@ -47,6 +47,8 @@ class SettingsManager:
     _FILENAME = "settings.yaml"
     _APPNAME = "Napari"
     _APPAUTHOR = "Napari"
+    application: ApplicationSettings
```

Comment:
these annotations are added so that you can get IDE autocompletion when you type `SETTINGS.application.<tab>`